jsscripts/InputMethodHandler.js

```diff
@@ -13,6 +13,8 @@ function debug(msg) {
   Logger.debug("InputMethodHandler.js -", msg);
 }
 
+ChromeUtils.import("resource://gre/modules/XPCOMUtils.jsm");
```

Comment:
```suggestion
const { XPCOMUtils } = ChromeUtils.import("resource://gre/modules/XPCOMUtils.jsm");
```

With esr78 you need to store return value value.